packages/wrangler/src/__tests__/versions/deployments/deployments.list.test.ts

```diff
@@ -39,63 +39,63 @@ describe("deployments list", () => {
-			Source:      Rollback
-			Message:     -
-			Version(s):  (10%) 10000000-0000-0000-0000-000000000000
-			                 Created:  2021-01-01T00:00:00.000Z
-			                     Tag:  -
-			                 Message:  -
-
-			             (90%) 20000000-0000-0000-0000-000000000000
-			                 Created:  2021-01-01T00:00:00.000Z
-			                     Tag:  -
-			                 Message:  -
-
-			Created:     2021-01-01T00:00:00.000Z
-			Author:      Jean-Luc-Picard@federation.org
-			Source:      Upload
-			Message:     -
-			Version(s):  (20%) 10000000-0000-0000-0000-000000000000
-			                 Created:  2021-01-01T00:00:00.000Z
-			                     Tag:  -
-			                 Message:  -
-
-			             (80%) 20000000-0000-0000-0000-000000000000
-			                 Created:  2021-01-01T00:00:00.000Z
-			                     Tag:  -
-			                 Message:  -
-
-			Created:     2021-02-02T00:00:00.000Z
-			Author:      Kathryn-Janeway@federation.org
-			Source:      Rollback
-			Message:     Rolled back for this version
-			Version(s):  (30%) 10000000-0000-0000-0000-000000000000
-			                 Created:  2021-01-01T00:00:00.000Z
-			                     Tag:  -
-			                 Message:  -
-
-			             (70%) 20000000-0000-0000-0000-000000000000
-			                 Created:  2021-01-01T00:00:00.000Z
-			                     Tag:  -
-			                 Message:  -
-
-			Created:     2021-02-03T00:00:00.000Z
-			Author:      Kathryn-Janeway@federation.org
-			Source:      Wrangler 🤠
-			Message:     -
-			Version(s):  (40%) 10000000-0000-0000-0000-000000000000
-			                 Created:  2021-01-01T00:00:00.000Z
-			                     Tag:  -
-			                 Message:  -
-
-			             (60%) 20000000-0000-0000-0000-000000000000
-			                 Created:  2021-01-01T00:00:00.000Z
-			                     Tag:  -
-			                 Message:  -
-			"
-		`);
+				"Created:     2021-01-01T00:00:00.000Z
+				Author:      Jean-Luc-Picard@federation.org
+				Source:      Upload
+				Message:     -
+				Version(s):  (20%) 10000000-0000-0000-0000-000000000000
+				                 Created:  2021-01-01T00:00:00.000Z
+				                     Tag:  -
+				                 Message:  -
+
+				             (80%) 20000000-0000-0000-0000-000000000000
+				                 Created:  2021-01-01T00:00:00.000Z
+				                     Tag:  -
+				                 Message:  -
+
+				Created:     2021-01-04T00:00:00.000Z
+				Author:      Jean-Luc-Picard@federation.org
+				Source:      Rollback
+				Message:     -
+				Version(s):  (10%) 10000000-0000-0000-0000-000000000000
+				                 Created:  2021-01-01T00:00:00.000Z
+				                     Tag:  -
+				                 Message:  -
+
+				             (90%) 20000000-0000-0000-0000-000000000000
+				                 Created:  2021-01-01T00:00:00.000Z
+				                     Tag:  -
+				                 Message:  -
```

Comment:
Why have these 2 deployments been switched in order without a change to the test? Not sure this snapshot should have updated

Reply:
We reversed the ordering in the list command - newest is now always at the bottom (in this case 4th is newer than the 1st)